### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by uchiiii** [salesforce/Merlion#152]
**Is your feature request related to a problem? Please describe.**
I would like to watch the trajectory of train loss, validation loss, parameters' settings etc.., but could not do that.

**Describe the solution you'd like**
Add feature to log these metrics to local file system or other services like mlflow. 

**Describe alternatives you've considered**

**Additional context**
pytorch lightening offers this functionality (https://bytepawn.com/automatic-mlflow-logging-for-pytorch.html), but it seems that refactoring is required.
